persistence-modules/jimmer/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>jpa-hibernate-cascade-type</artifactId>
+  <properties>
+    <jimmer.version>0.9.37</jimmer.version>
```

Comment:
The latest version seems to be 0.9.48
https://mvnrepository.com/artifact/org.babyfish.jimmer/jimmer-spring-boot-starter

Reply:
Upgraded to 0.9.81